lib/options/videosize.js

```diff
@@ -6,17 +6,35 @@
  */
 
 
+function getScaleFilter(output) {
+  if (output.video.get('-vcodec').length > 1) {
+    let codec = output.video.get('-vcodec')[1];
```

Comment:
Such condition would not work in case of the video codec option is not the first one in the list.

The following approach works correct:
```js
if (!!output.video.find('-vcodec', 1)) {
    let codec = output.video.find('-vcodec', 1)[0];
```